test/e2e/scenario/recorder/shadowDom.scenario.ts

```diff
@@ -104,6 +134,33 @@ describe('recorder with shadow DOM', () => {
       expect(textNode?.textContent).toBe('toto')
     })
 
+  if (isAdoptedStyleSheetsSupported()) {
+    createTest('can record fullsnapshot with adoptedStylesheet')
+      .withRum({ defaultPrivacyLevel: 'allow', enableExperimentalFeatures: ['record_shadow_dom'] })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is `defaultPrivacyLevel: 'allow'` needed for this test?

Reply:
no, not really, I will rely on the default